### ISSUE TITLE
Program type already present: com.urbanairship.BuildConfig

### ISSUE THREAD

**Issue by uzairaslam1993** [urbanairship/android-library#77]
after upgrading urban airship from version 6 to 9 , my app is not compiling i am getting this issue 

Program type already present: com.urbanairship.BuildConfig

using urbanairship sdk 9.1.0

**Comment by uzairaslam1993** [comment]
for some reason i cannot share the build.gradle file. Already cleaned the project. When i downgrade to 9.0.2 , it seems to be working

**Comment by rlepinski** [comment]
Could you list the UA dependencies ?

**Comment by rlepinski** [comment]
@uzairaslam1993 I assume when you updated you are now pulling in `urbanairship-fcm` instead of `urbanairship-sdk` right? Do you have another library that is pulling in an older version of the `urbanairship-sdk` version? Run `./gradlew app:dependencies` (replace `app` with whatever your app module is)

**Comment by rafagcamargo** [comment]
I was having the same problem and was able to fix it by specifying the **core** and **gcm** packages:
```
implementation "com.urbanairship.android:urbanairship-core:$UrbanAirshipVersion"
implementation "com.urbanairship.android:urbanairship-gcm:$UrbanAirshipVersion"
```
Before I had:
```
implementation "com.urbanairship.android:urbanairship-sdk:$UrbanAirshipVersion"
```

**Comment by uzairaslam1993** [comment]
implementation 'com.urbanairship.android:urbanairship-sdk:9.0.2' ,, this is what i am using now and not giving me the error. When I used 9.1.0 for sdk and gcm, I got the error.

**Comment by rlepinski** [comment]
I see the problem, will fix in 9.1.2.

`urbanairship-sdk` just pulls in `urbanairship-gcm` and `urbanairship-adm`. Both `urbanairship-sdk` and `urbanairship-gcm` are deprecated and will eventually be dropped. We want everyone to move to pulling in specific push providers and the new `urbanairship-fcm` package over `urbanairship-gcm`.  See [fcm migration guide]( https://github.com/urbanairship/android-library/blob/master/documentation/migration/migration-guide-fcm.md) for more info.
If you want to continue to use what the `urbanairship-sdk` package pulls in, remove the `urbanairship-sdk` module and instead just import:
```
implementation "com.urbanairship.android:urbanairship-adm:9.1.1"
implementation "com.urbanairship.android:urbanairship-gcm:9.1.1"
```

**Comment by uzairaslam1993** [comment]
Thanks @rlepinski for your response. Yes we are looking forward to migrate and this will be next on our agenda.

**Comment by rlepinski** [comment]
urbanairship-sdk package fixed in 9.2.0

**Comment by Mahdiazadbar** [comment]
You can see this response
https://stackoverflow.com/a/26531074/4097495
in my case solve with this suggestion.
my app and my module have same package name
